docs/other.md

```diff
@@ -0,0 +1,185 @@
+on this topic.
+4. Modify longitudinal, neighbor, and diversity sampling parameters as
+desired. If you end up experimenting with different values for these
+parameters, which we encourage, we would love to hear about your findings. Be
+aware that increasing the `*_PERCENT_ID` parameters will increase the runtime
+of your analysis, and decreasing those values will decrease the runtime of your
+analysis.
+
+(alignment-mask)=
+## Pre-computed alignment mask
+
+We provide support for masking alignments with the pre-computed mask
+provided [here](https://github.com/W-L/ProblematicSites_SARS-CoV2). Like the
+rest of our {ref}`alignment and phylogenetics workflow <downstream>`,
+this is experimental. Let us know how it works for you. It would be very useful
+to hear about how your final results (e.g., a phylogenetic tree) computed
+using this mask compares to one created with your own custom mask.
+
+(removing-sequences)=
+## Removing sequences present in both focal and context sequence collections
+
+As many researchers are rapidly submitting their sequence data to public
+repositories, sometimes your focal sequences may be present in context
+sequences that you obtain from a public repository. You can use QIIME 2 to
+remove those sequences from your context sequence collection.
+
+First, visualize your focal sequence and context sequence collections with
+QIIME 2 to confirm that you know what QIIME 2 considers to be the sequence
+identifiers. You can do this by running the following two commands:
+
+```
+qiime feature-table tabulate-seqs \
+ --i-data focal-seqs.qza \
+ --o-visualization focal-seqs.qzv
+
+qiime feature-table tabulate-seqs \
+ --i-data context-seqs.qza \
+ --o-visualization context-seqs.qzv
+```
+
+Then, load the two visualizations with [QIIME 2 View](https://view.qiime2.org).
+The values in the _Feature ID_ column are the sequence ids. If the ids of the
+sequences that you want to remove are the same across your two data sets, you
+can follow the steps under {ref}`shared-identifiers`. If the identifiers you
+want to remove are not the same across your two data sets, or you're not sure,
+follow the steps under {ref}`differing-identifiers`.
+
+(shared-identifiers)=
+### Shared identifiers
+
+If your sequence identifiers are identical across your focal and context
+sequence collections, you can remove context sequences that are also focal
+sequences as follows:
+
+```
+qiime feature-table filter-seqs\
+ --i-data context-seqs.qza \
+ --m-metadata-file focal-seqs.qza \
+ --p-exclude-ids \
+ --o-filtered-data filtered-context-seqs.qza
+```
+
+All of your downstream analysis steps should then use the resulting
+`filtered-context-seqs.qza` file.
+
+
+Alternatively, if you want to remove these sequences from your focal sequence
+collection, you could do this as follows:
+
+```
+qiime feature-table filter-seqs\
+ --i-data focal-seqs.qza \
+ --m-metadata-file context-seqs.qza \
```

Comment:
So I have learned from the benchmarking that this in practice make provenance absolutely massive. We might want to find a new approach here.

Reply:
Fixed this by updated to pass metadata as metadata instead of sequences as metadata.